project_resource_calendar/views/instrument.xml

```diff
@@ -49,18 +48,16 @@
                         <group>
                             <field name="sku"/>
```

Comment:
You should add the photo field. You can check here the format: https://github.com/savoirfairelinux/project-addons/pull/73/commits/0394a61b9ef628d9dcbc7c265c148d44d3181f23

Reply:
@jananjoy you should put 1 main photo after the name of the instrument and 4 photos in the photos page